packages/web-components/src/components/va-breadcrumbs/test/va-breadcrumbs.e2e.ts

```diff
@@ -0,0 +1,79 @@
+import { newE2EPage } from '@stencil/core/testing';
+import { axeCheck } from '../../../testing/test-helpers';
+
+describe('va-breadcrumbs', () => {
```

Comment:
Can we some tests to cover that `<li>` and `<a>` tags are rendered appropriately?